### ISSUE TITLE
Image alt text issue

### ISSUE THREAD

**Issue by aitayn** [HarryChen0506/react-markdown-editor-lite#64]
You are adding alt text to img tag which is selected on editor. It should be the name of image which is uploading. Can I get it fixed as soon as possible?

**Comment by HarryChen0506** [comment]
can you submit a pr for it  if you are  urge？

**Comment by HarryChen0506** [comment]
fixed in version 0.5.2
